Cql/Cql.CqlToElm/ModelProvider.cs

```diff
@@ -14,10 +14,10 @@ public static class ModelProvider
         /// Returns the <see cref="ModelInfo"/> for a model, given the model's uri and version.
         /// </summary>
         /// <exception cref="ArgumentException">There is no model with the given uri.</exception>
-        public static Model.ModelInfo GetModelFromUri(this IModelProvider provider, string uri, string? version = null) =>
+        public static Model.ModelInfo? GetModelFromUri(this IModelProvider provider, string uri, string? version = null) =>
```

Comment:
In public interfaces, I prefer using `TryXXXX` instead of returning null to signal failure.

Reply:
Actually....there is already a TryGetModelFromUri that does not throw - so I think this change does not make sense, just call that method instead.

Reply:
And then this method became unused, so I have removed it.